xml/schema/Core/LocationType.xml

```diff
@@ -11,6 +11,7 @@
     <add>civicrm/admin/locationType/edit?action=add&amp;reset=1</add>
     <update>civicrm/admin/locationType/edit?action=update&amp;id=[id]&amp;reset=1</update>
     <delete>civicrm/admin/locationType/edit?action=delete&amp;id=[id]&amp;reset=1</delete>
+    <browse>civicrm/admin/locationType</browse>
```

Comment:
@aydun it's already here!

Reply:
But good to see the script gives the exact same output as the hand-added one :)

Reply:
Grr!  The script was supposed to recognise that...  it does now.